client/constants.py

```diff
@@ -25,10 +25,33 @@ class UrlTypes(object):
     YOUTUBE = "youtube"
 
     special_urls = {
-        WIKI: "en.wikipedia.org",
+        WIKI: "wikipedia.org",
         YOUTUBE: "youtube.com"
     }
 
     @staticmethod
     def get_special_url(url_type):
         return UrlTypes.special_urls.get(url_type)
+
+
+class JsonFields(object):
+    COUNT = "count"
+    DATA = "data"
+    DESC = "description"
+    IFRAME = "iframe"
```

Comment:
not sure if we should have both iframe and media, our hopes is that the developer does not have to do additional logic in order to display the metadata.
So for something like a video, we should instead of a field inside media that's something like "view-type" : "iframe".
How we structure this is open to discussion